index.js

```diff
@@ -1,4 +1,4 @@
 'use strict';
 module.exports = function () {
-	return /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g;
+	return /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-RZcf-nqry=><]/g;
```

Comment:
Can you instead change it to `A-PRZcf`? You're introducing `Q` as well, which hasn't been brought up as a valid escape sequence.